divio_cli/cli.py

```diff
@@ -207,12 +207,15 @@ def sort_projects(items):
 
 @project.command(name='deploy')
 @click.argument('stage', default='test')
+@click.option(
+    '--backup/--no-backup', 'backup', default=False
+)
```

Comment:
Could you move the options to before the arguments? So they are in the same canonical order as the CLI expects them?

Also, 'backup' is probably not needed.